src/operator/pooling-inl.h

```diff
@@ -47,6 +49,11 @@ struct PoolingParam : public dmlc::Parameter<PoolingParam> {
     .add_enum("avg", pool_enum::kAvgPooling)
     .add_enum("sum", pool_enum::kSumPooling)
     .describe("Pooling type to be applied.");
+ 
+    DMLC_DECLARE_FIELD(pad_convention).set_default(pool_enum::kValid)
+    .add_enum("same", pool_enum::kSame)
```

Comment:
change same to "full" or "compatible" and add document describing the difference (round up or down, discard extra or pad zero) and say that use full for compatibility with caffe